### PR TITLE
change fasthttp to origin net/http

### PR DESCRIPTION
**Pull request by wangkirin**
Signed-off-by: WangQilin <qilin.wang@huawei.com>

### CODE REVIEW

**Comment by tianxiaoliang** [comment]
tracing should only accept map as Router component, because chassis is mean to support any protocol, plz refer to router.
delete all fasthttp files in lib

File: core/tracing/carrier.go [hunk 1]
```diff
@@ -3,7 +3,7 @@ package tracing
 import (
 	"github.com/ServiceComb/go-chassis/client/rest"
 
-	"github.com/ServiceComb/go-chassis/third_party/forked/valyala/fasthttp"
+	"github.com/valyala/fasthttp"
```
Comment (tianxiaoliang) [inline core/tracing/carrier.go]: remove this dependency

Comment (wangkirin) [reply core/tracing/carrier.go]: already fixed

**Comment by wangkirin** [comment]
@tianxiaoliang  
adjustion have done ,  please take a look

File: core/tracing/carrier.go [hunk 2]
```diff
@@ -26,7 +24,7 @@ func (r RestClientHeaderWriter) Set(key, val string) {
 
 // FasthttpHeaderCarrier fast http heaer carrier
 type FasthttpHeaderCarrier struct {
```
Comment (tianxiaoliang) [inline core/tracing/carrier.go]: Change to HeaderCarrier

Comment (wangkirin) [reply core/tracing/carrier.go]: DONE

**Pull request merged**